planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -1486,6 +1486,9 @@ ObstacleAvoidancePlanner::alignVelocity(
   size_t prev_begin_idx = 0;
   for (size_t i = 0; i < fine_traj_points_with_vel.size(); ++i) {
     const auto truncated_points = points_utils::clipForwardPoints(path_points, prev_begin_idx, 5.0);
```

Comment:
```suggestion
    auto truncated_points = points_utils::clipForwardPoints(path_points, prev_begin_idx, 5.0);
```

Reply:
I think `truncated_points.assign(path_points.begin(), path_points.end());` is O
but I found an odd behavior please wait for murooka-san
https://user-images.githubusercontent.com/65527974/179894007-63cfcd49-b4a1-47b3-8e65-1be2bfa0add3.mp4